src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -431,7 +450,7 @@ public BanditResult getBanditAction(
     }
   }
 
-  private Map<String, String> buildLogMetaData() {
+  private Map<String, String> buildLogMetaData(boolean isConfigObfuscated) {
```

Comment:
Are we passing this through because it's possible the initial configuration may be passed differently than whether it expects obfuscated configurations? I'm ok with this but having trouble thinking when the two would be out of sync and wondering if this is adding complexity vs just using what is passed into the constructor.

Reply:
If initial configuration is not obfuscated, there's going to be a change of obfuscation'ness on Android after the configuration is changed from the server. Because this may happen concurrently mid-evaluation, evaluation must hold and use a stable `Configuration` reference that doesn't change under it